templates/education-internships.html

```diff
@@ -15,18 +15,63 @@ <h3>Internships</h3>
 
 <div>
 
+        <div class="text-center">
+            <figure><img src="{{ url_for('static', filename='img/HLT_Footer.jpeg') }}" alt="HLT Banner" style="width: 100%;"></figure>
+        </div>
+
     <p>
-        ...
+        The Chair in Computational Lexicology liaises with a range of companies in the Netherlands and abroad.
+        This involves cooperation projects, internships, participation in minor programs for employees, advice
+        and consultation. Recent contacts are: <a href="https://www.accenture.com/nl-en" target="_blank">Accenture</a>,
+        <a href="https://www.bol.com/nl/nl/" target="_blank">bol.com</a>, <a href="https://www.cbs.nl/en-gb" target="_blank">Statistics Netherlands (CBS)</a>,
+        <a href="https://www2.deloitte.com/nl/nl.html" target="_blank">Deloitte</a>, <a href="https://home.kieskompas.nl/en/" target="_blank">Election Compass (Kieskompas)</a>,
+        <a href="https://www.kb.nl/en" target="_blank">National Library of the Netherlands (Koninklijke Bibliotheek)</a>, <a href="https://www.lexisnexis.co.uk/" target="_blank">LexisNexis</a>,
+        <a href="https://home.mediquest.nl/" target="_blank>">MediQuest</a>, <a href="https://www.forensicinstitute.nl/" target="_blank">Netherlands Forensic Institute (NFI) Ministry of Justice and Security</a>,
+        <a href="https://nlterm.org/" target="_blank">NL-Term</a>, <a href="https://olery.com/" target="_blank">Olery</a>, <a href="https://www.oracle.com/" target="_blank">Oracle</a>
+        , <a href="https://www.synerscope.com/" target="_blank">Synerscope</a>, <a href="https://www.trivago.com/" target="_blank">Trivago</a>,
```

Comment:
the comma at the start of line 31 can better be placed at the end of line 30 (this creates an extra whitespace now)